config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
   root 'users#index'
-
-  resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
-  end
+  get '/users', to: 'users#index', as: :users
+  get '/users:id', to: 'users#show', as: :user
+  get 'users:user_id/posts', to: 'posts#index', as: :user_posts
+  get 'users:user_id/posts:id', to: 'posts#show', as: :user_post
```

Comment:
- [ ] Please add a `/` before the id part of the url for the highlighted lines above. For example 👇 
```ruby
get '/users/:user_id/posts/:id', to 'posts#show', as: :user_post
```